SPECS/gcc/gcc.spec

```diff
@@ -230,6 +436,8 @@ $tests_ok
 %exclude %{_lib64dir}/libstdc++*
 %exclude %{_lib64dir}/libsupc++*
 
+%do_exclude aarch64-linux-gnu %{build_aarch64}
```

Comment:
Same comment as `do_files`, need some explanation.

Reply:
Added a comment. Thank you, Daniel.